File: 9_bitStrings.cpp
#include <bits/stdc++.h>
using namespace std;

unsigned long long MOD = 1000000007;

unsigned long long bitStrings(int &n)
{
    unsigned long long answer = 1;
    while (n--)
    {
        answer = (answer % MOD) * 2;
    }
    return answer % MOD;
}

int main()
{
    int n;
    cin >> n;
    cout << bitStrings(n) << endl;
}

File: 9_bitStrings.py
def bitStrings(n: int):
    answer = 1
    for i in range(n):
        answer = (answer % (1e9 + 7)) * 2

    return answer % (1e9 + 7)


if __name__ == "__main__":
    n = int(input())
    print(int(bitStrings(n)))
